modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -5247,6 +5455,62 @@ GDScriptParser::DataType GDScriptAnalyzer::type_from_property(const PropertyInfo
 			}
 			elem_type.is_constant = false;
 			result.set_container_element_type(0, elem_type);
+		} else if (p_property.type == Variant::DICTIONARY && p_property.hint == PROPERTY_HINT_DICTIONARY_TYPE) {
+			// Check element type.
+			PackedStringArray elem_type_names = (p_property.hint_string).split(";");
```

Comment:
```suggestion
			PackedStringArray elem_type_names = p_property.hint_string.split(";");
```

Reply:
Updated to instead utilize `get_slice` like the [above suggestion](https://github.com/godotengine/godot/pull/78656#discussion_r1738626887).